src/Data/Repositories/LiquidityRuleRepository.cs

```diff
@@ -105,7 +111,7 @@ public async Task<ICollection<LiquidityRule>> GetByNodePubKey(string nodePubKey)
             .Include(x=> x.Wallet)
             .ThenInclude(x=> x.InternalWallet)
             .Include(x=> x.Channel)
-            .Where(x=> x.Node.PubKey == nodePubKey).ToList();
+            .Where(x=> x.Node.PubKey == nodePubKey && x.Channel.IsAutomatedLiquidityEnabled).ToList();
```

Comment:
No need to change this now but I would have written a different method or perhaps change the name of this one (or even better, pass isAutomatedLiquidityEnabled as a boolean with the method signature) to reflect its new purpose. Someone might want to get nodes by pubkey regardless of it having automated liquidity enabled or not

Reply:
Agree but right now we would not need never a liquidity rule which is disabled in liquidator, so that's why is hardcoded